pom.xml

```diff
@@ -272,20 +301,29 @@
         <maven.checkstyle.plugin.version>3.1.0</maven.checkstyle.plugin.version>
         <maven.spotbugs.plugin.version>4.2.3</maven.spotbugs.plugin.version>
         <spotbugs.annotations.version>4.7.3</spotbugs.annotations.version>
+        <maven.bundle.plugin.version>3.2.0</maven.bundle.plugin.version>
         <com.h3xstream.findsecbugs.version>1.12.0</com.h3xstream.findsecbugs.version>
         <commons.logging.version>1.2</commons.logging.version>
-        <axiom.wso2.version>1.2.11-wso2v16</axiom.wso2.version>
+        <axiom.wso2.version>1.2.11-wso2v28</axiom.wso2.version>
         <commons-lang.version>3.4</commons-lang.version>
-        <json.wso2.version>3.0.0.wso2v4</json.wso2.version>
-        <com.googlecode.json.version>1.1.1</com.googlecode.json.version>
+        <json-smart.version>2.4.10</json-smart.version>
         <commons-collections.version>4.4</commons-collections.version>
         <org.testng.version>7.7.0</org.testng.version>
         <jacoco.version>0.8.6</jacoco.version>
         <commons-dbcp.version>1.4</commons-dbcp.version>
         <orbit.version.h2.engine>1.2.140.wso2v3</orbit.version.h2.engine>
         <mockito.version>1.10.19</mockito.version>
         <powermock.version>1.7.1</powermock.version>
```

Comment:
<powermock.version> is already defined, shall we remove this one?
```suggestion
```

Reply:
Check mockito.version as well

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/3/commits/08d281c66fc90f979dcae36a8c8baedf99ab9aa9